express/code/blocks/how-to-v2/how-to-v2.css

```diff
@@ -0,0 +1,216 @@
+}
+
+main .how-to-v2 .milo-video,
+main .how-to-v2 picture img {
+  border-radius: 16px;
+  overflow: hidden;
+}
+
+main .how-to-v2 em {
+  font-style: normal;
+  background: linear-gradient(140.08deg, #FF4DD2 67.54%, #FF993B 76.42%);
+  background-size: 108% 108%;
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  background-clip: text;
+  font-weight: 900;
+}
+
+main .how-to-v2 .steps-content {
+  display: flex;
+  flex-direction: column;
+  padding: 0 24px;
+  position: relative;
+}
+
+main .how-to-v2 .steps-content .steps-content-backg {
+  position: absolute;
+  width: 100%;
+  top: -24vw;
+  z-index: -1;
+  height: 100%;
+  transform: scale(0.9);
+}
+
+main .how-to-v2 ol.steps {
+  margin: 6px 0 0;
+  padding: 0;
+}
+
+main .how-to-v2 ol.steps .step .step-content h3:not(:has(picture), :has(.milo-video)):hover {
+  background-color: var(--color-gray-100);
+  margin: -8px 0 -8px -16px; 
+  padding: 8px 0 8px 16px;
+}
+
+main .how-to-v2 .video-container {
+  margin-top: 20px;
+}
+
+main .how-to-v2 .image-container {
+  margin-top: 20px;
+  width: 100%; 
+  overflow: hidden;
+}
+
+main .how-to-v2 .image-container img,
+main .how-to-v2 .video-container > * {
+  border-radius: 16px;
+  max-width: 400px;
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+}
+
+main .how-to-v2 .video-container > * {
+  margin-left: auto;
+  margin-right: auto;
+  height: unset;
+}
+
+main .how-to-v2 {
+  /* note we don't set margin here because background gradient has to go past into this container */
+  margin: 0;
+  max-width: unset;
+}
+
+@media (min-width: 768px) {
+
+  main .how-to-v2 h2 {
+    font-size: 28px;
+  }
+
+  main .how-to-v2 {
+    max-width: 950px;
+    margin: auto;
+  }
+
+  main .how-to-v2 .steps-content {
+    display: flex;
+    flex-direction: row;
+    padding: 0 24px;
+    justify-content: center;
+    align-items: center;
+  }
+
+  main .how-to-v2 .steps-content > * {
+    flex: 1 1 50%;
+    box-sizing: border-box;
+  }
+
+  main .how-to-v2 .image-container,
+  main .how-to-v2 .video-container {
+    align-items: center;
+    display: inline-grid;
+  }
+
+  main .how-to-v2 .image-container img {
+    height: unset;
+  }
+
+  main .how-to-v2 .steps-content .steps {
+    padding-left: 20px;
+  }
+
+  main .how-to-v2 .steps-content .steps-content-backg {
+    left: -90px;
+    width: 67%;
+    transform: scale(0.9);
+    top: -68px;
+  }
+}
+
+@media (min-width: 1280px) {
+
+  main .how-to-v2 h2 {
+    font-size: 36px;
+  }
+
+  main .how-to-v2 {
+    max-width: 1300px;
+    margin: auto;
+  }
+
+  main .how-to-v2 .image-container img,
+  main .how-to-v2 .video-container > * {
+    max-height: 300px;
+    max-width: 533px;
+  }
+
+  main .how-to-v2 .image-container {
+    display: inline-block;
+  }
+
+  main .how-to-v2 .steps-content {
+    margin-top: 30px;
+  }
+
+  main .how-to-v2 ol li h3 {
+    text-align: left;
+    font-size: 22px;
+    line-height: 28.6px;
+    font-weight: 700;
+  }
```

Comment:
Nit:

`Text-align` and `font-weight` are already declared [on line 40](https://github.com/adobecom/express-milo/pull/128/files#diff-15d73b7a4aea9f88084ea0e5485215ed1a623784029f760c15ced036077e6c8cR40-R45). Do they need to be called again for any reason?